habitat_baselines/config/default.py

```diff
@@ -37,6 +37,12 @@
 _C.LOG_FILE = "train.log"
 _C.CHECKPOINT_INTERVAL = 50
 # -----------------------------------------------------------------------------
+# EVAL CONFIG
+# -----------------------------------------------------------------------------
+_C.EVAL = CN()
+_C.EVAL.USE_VAL = True
```

Comment:
Maybe better make it configurable, then we can use different splits for evaluation, but have "val" as default:
```suggestion
_C.EVAL.SPLIT = "val"
```

Reply:
That seems like a better way to do it!